packages/langium/src/test/langium-test.ts

```diff
@@ -550,18 +551,29 @@ export interface ValidationResult<T extends AstNode = AstNode> extends AsyncDisp
     document: LangiumDocument<T>;
 }
 
-export function validationHelper<T extends AstNode = AstNode>(services: LangiumCoreServices): (input: string, options?: ParseHelperOptions) => Promise<ValidationResult<T>> {
+export type ValidationHelperOptions = ParseHelperOptions & { failOnParseErrors?: true };
```

Comment:
Suggestion: Generalize the property type to `boolean` to enable forwarding the option from some other source (so we don't know whether it's set to true or false).

Other suggestion: rename to `failOnParsingErrors` to be consistent with the constant from DocumentValidator.